Rakefile

```diff
@@ -12,7 +12,7 @@ task :default => [:test]
 
 def run_specs paths
   format = ENV['VERBOSE'] ? '--format documentation ' : ''
-  sh "rspec -w #{format}#{paths.join ' '}"
+  sh "bundle exec rspec #{format}#{paths.join ' '}"
```

Comment:
Is removal of `-w` intentional?

Reply:
The output of the run is much less noisy:
https://travis-ci.org/pry/pry/jobs/301676683